pkg/kubectl/pdb.go

```diff
@@ -167,15 +167,6 @@ func (s *PodDisruptionBudgetV2Generator) StructuredGenerate() (runtime.Object, e
 		return nil, err
 	}
 
-	if len(s.MaxUnavailable) == 0 && len(s.MinAvailable) == 0 {
-		s.MinAvailable = "1"
-
-		// This behavior is intended for backward compatibility.
-		// TODO: remove in Kubernetes 1.8
-		fmt.Fprintln(os.Stderr, "Deprecated behavior in kubectl create pdb: Defaulting min-available to 1. "+
```

Comment:
https://k8s-gubernator.appspot.com/build/kubernetes-jenkins/pr-logs/pull/53047/pull-kubernetes-e2e-kops-aws/50520
```
W0926 16:37:37.741] # k8s.io/kubernetes/pkg/kubectl
W0926 16:37:37.742] pkg/kubectl/pdb.go:21: imported and not used: "os"
W0926 16:37:49.206] !!! [0926 16:37:49] Call tree:
W0926 16:37:49.207] !!! [0926 16:37:49]  1: /go/src/k8s.io/kubernetes/hack/lib/golang.sh:720 kube::golang::build_binaries_for_platform(...)
W0926 16:37:49.209] !!! [0926 16:37:49]  2: hack/make-rules/build.sh:27 kube::golang::build_binaries(...)
W0926 16:37:49.213] !!! [0926 16:37:49] Call tree:
W0926 16:37:49.215] !!! [0926 16:37:49]  1: hack/make-rules/build.sh:27 kube::golang::build_binaries(...)
W0926 16:37:49.219] !!! [0926 16:37:49] Call tree:
W0926 16:37:49.221] !!! [0926 16:37:49]  1: hack/make-rules/build.sh:27 kube::golang::build_binaries(...)
```
need to remove the "os" import since you removed the only use of the package

Reply:
Got it and Thanks

Reply:
Done